pulseapi/utility/newsletters/signup.py

```diff
@@ -1,112 +0,0 @@
-from rest_framework.decorators import api_view, parser_classes, permission_classes
```

Comment:
change history of this file: https://github.com/mozilla/network-pulse-api/commits/master/pulseapi/utility/newsletters/signup.py